check_node_version.js

```diff
@@ -0,0 +1,6 @@
+const semver = require('semver')
+const { engines } = require('./package')
+const version = engines.node
+if (!semver.satisfies(process.version, version)) {
+  throw new Error(`The current node version${process.version} does not satisfy the required version ${version} .`)
+}
```

Comment:
Please move this file to `scripts` folder and keep the filename in camelCase instead of snake_case